setup.cfg

```diff
@@ -1,7 +1,7 @@
 [metadata]
 description-file = README.md
 [tool:pytest]
-addopts = --doctest-modules
+addopts = --doctest-modules --ignore=featuretools/tests/plugin_tests/featuretools_plugin
```

Comment:
During the test collection in pytest, an error is raised if the test package (`featuretools_plugin`) is in the path.

Reply:
Can we similarly have coverage ignore the test package?

Reply:
Yes, updated config in code coverage.